taxi/views.py

```diff
@@ -34,14 +35,24 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
-class CarListView(LoginRequiredMixin, generic.ListView):
-    model = Car
-    paginate_by = 5
-    queryset = Car.objects.all().select_related("manufacturer")
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
 
 
-class CarDetailView(LoginRequiredMixin, generic.DetailView):
-    model = Car
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Manufacturer
+    template_name = "taxi/manufacturer_format_confirm_delete.html"
```

Comment:
The template name `taxi/manufacturer_format_confirm_delete.html` seems to have a typo. It should likely be `taxi/manufacturer_confirm_delete.html` to follow Django's convention and match the expected template naming pattern.